css/nixos-site.css

```
@@ -99,6 +101,11 @@ pre {
     content: " [nix-shell]$ ";
 }
 
+/* mobile friendly */
+div.main {
+    word-wrap: break-all;
```

```suggestion
    overflow-wrap: break-word;
```

@samueldr this seems to do the job: https://jsfiddle.net/4ouc90p2/

---

Yes, just tested, this is fine. It might cause the same weird break in a word if somehow we had one of those big impossibly long one where there is no other place to break, but at that point it's a non-issue.